### PR TITLE
NewRelic Capacitor Plugin

### PR DESCRIPTION
**Pull request by ndesai-newrelic**
Newrelic Capacitor Plugin:

@a-sassman  we can discuss more on this.

### CODE REVIEW

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+# iOS apps
+ionic capacitor run ios
+```
+
+## API calls [#api-calls]
+
+Our Capacitor Plugin utilizes the same API calls as our iOS and Android SDK agents. Please refer to [New Relic iOS SDK doc](/docs/mobile-monitoring/new-relic-mobile-ios/ios-sdk-api) or [Android SDK](/docs/mobile-monitoring/new-relic-mobile-android/android-sdk-api).
+
+For errors specific to React Native, you can call `NewRelicCapacitorPlugin.recordError(error)` as follows:
```
Comment (a-sassman) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:113]: @ndesai-newrelic Is there supposed to be a code sample here?

Comment (kthai-nr) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:113]:
````suggestion
For errors specific to Capacitor, you can call `NewRelicCapacitorPlugin.recordError` as follows:
```ts
    try {
      throw new Error('Example error message');
    } catch (error: any) {
      NewRelicCapacitorPlugin.recordError({
        name: error.name,
        message: error.message,
        stack: error.stack,
        isFatal: false,
      });
    }
```
````

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+* Track promise rejection (an unhandled exception in JavaScript)
+* Track warnings and errors with console logs
+* Capture interactions and their sequences
+* Track user sessions
+
+For more information, see [New Relic Capacitor Plugin on Github](https://github.com/newrelic/newrelic-capacitor-plugin).
+
+## Requirements [#requirements]
+
+Before you install the React Native agent, ensure your app meets these requirements:
```
Comment (kthai-nr) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:34]:
```suggestion
Before you install the Capacitor agent, ensure your app meets these requirements:
```

**Review by ghost — DISMISSED** [review]

**Comment by a-sassman** [comment]
@ndesai-newrelic Is this ready to be published?

**Comment by ndesai-newrelic** [comment]
@a-sassman  not yet, i will inform you once we are ready.

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+* Android API 24+
+* iOS 10
+* [iOS native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
+* [Android native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
+
+
+## Install
+
+```bash
+npm install newrelic-capacitor-plugin
```
Comment (kthai-nr) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:45]: I've changed how the npm package should be scoped:
```suggestion
npm install @newrelic/newrelic-capacitor-plugin
```

File: src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+npx cap sync
+```
+## Ionic Capacitor Setup
+
+Now open your `index.tsx` and add the following code to launch NewRelic (don't forget to put proper application tokens):
+
+``` tsx
+import { NewRelicCapacitorPlugin } from 'newrelic-capacitor-plugin';
```
Comment (kthai-nr) [inline src/content/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor.mdx:53]: Another change caused by the new npm scope:
```suggestion
import { NewRelicCapacitorPlugin } from '@newrelic/newrelic-capacitor-plugin';
```

**Review by ghost — DISMISSED** [review]

**Review by ghost — APPROVED** [review]

**Comment by a-sassman** [comment]
@ndesai-newrelic This will go live with tomorrow's 9am publish of the docs site.

**Pull request merged**